app.js

```diff
@@ -164,6 +164,20 @@ if (config.ld_library_path.indexOf('api') != -1) {
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({extended: true}));
 
+
+// check node type
+function check_type_node(){
```

Comment:
Notice that the `fs.readFileSync()` or the `toJson()` methods may throw errors, and they wouldn't be caught here with this implementation. To prevent this, wrap the content of the function into a `try catch` block:
```js
function check_type_node(){
  try {
      // function body
    } catch (err) {
    // handle error here
    }
}